kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -90,6 +90,7 @@ public ApiClient() {
         // Setup authentications (key: authentication name, value: authentication).
         authentications = new HashMap<String, Authentication>();
         authentications.put("BearerToken", new ApiKeyAuth("header", "authorization"));
+        authentications.put("BasicAuth", new HttpBasicAuth());
```

Comment:
It's in the ignore list https://github.com/kubernetes-client/java/pull/61/files/13b2813174966eb5ca6ee1cff700531e5492273f#diff-ea99bc9c9c08dd8a42e296305e0e9c48R7